modulefiles/wflow_hercules.lua

```diff
@@ -7,7 +7,7 @@ whatis([===[Loads libraries needed for running RRFS workflow on Hercules ]===])
 
 load("contrib")
 load("rocoto")
-load("wget")
+--load("wget")
```

Comment:
why we need to uncomment load("wget")?

Reply:
I saw your commit message now.
I think we will need to load wget at the moment.

Reply:
I get the following error when trying to load wget:

```
Lmod has detected the following error:  The following module(s) are unknown: "wget"
```

Because of this error, conda is not loaded, so I cannot do `conda activate workflow_tools`. Removing wget fixes the error, and does not appear to negatively impact the workflow generation process.

Not sure if you're also getting this error?

Reply:
@ShawnMurdzek-NOAA Which task generates this error?  
My non-DA tests did not have this error.

Reply:
The error occurs when loading the environment to generate workflow. E.g.,

```
module use modulefiles
module purge
module load wflow_hercules
```

I did some more digging and it looks like the intel compiler from spack-stack needs to be loaded first before loading wget. I'll add that change to this PR.

Reply:
I did not get any errors in generating the workflow.  I cannot do a further test right now since Hercules is slow today.
However, you don't need to "module purge" before you load wflow_hercules unless you already load some "unhappy" modules in your environment through bashrc or so.

Reply:
Is it possible that you loaded build_hercules_intel before loading wflow_hercules? That would also avoid the error because intel is loaded in build_hercules_intel.

Reply:
@ShawnMurdzek-NOAA Thanks for your Slack message! 
Yes, I can confirm that on Hercules wget needs to be loaded after intel.

However, for a "real" RRFS workflow, we have to do the following two `module load `before generating a workflow:
```
module load build_<machine>
module load wflow_<machine>
```
The reason is that we need to link the exactly same CRTM_FIX as the build environment. This information was documented somewhere in the "regional_workflow" era but got lost when we transitioned to rrfs-workflow. 

Thanks for bringing up this issue and I will ask Chan-Hoo at EMC to update the "Quick Start Guide" to reflect this. 
And we will keep this lua file unchanged. Is this good for you?

Reply:
OK, it looks like I did not pay much attention to the rewritten wflow lua file. 
They now explicitly put the CRTM version information into the wflow lua file.

In this sense, your change is valid. I will go ahead to merge this PR.  Thanks!

Reply:
Leaving the lua file unchanged for now sounds good! I noticed that CRTM is loaded within wflow_hercules... does that mean that we do not need to load build_hercules_intel first?